### PR TITLE
Home Page Changes AB#10951

### PR DESCRIPTION
**Pull request by vaishnav-canarys**
Home Page Changes [AB#10951](https://dev.azure.com/rubdevopson/c53ed56f-c130-470b-a6ae-3c4a93266530/_workitems/edit/10951)

### CODE REVIEW

**Comment by sriplayground** [comment]
Are these the only changes?

File: Tailspin.SpaceGame.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -12,8 +12,8 @@
 </style>
 <section class="intro">
     <div class="container">
-        <img class="title" src="/images/space-game-title.svg" alt="Space Game Adventure">
-        <p>Session for DevOps</p>
```
Comment (vaishnav-canarys) [inline Tailspin.SpaceGame.Web/Views/Home/Index.cshtml:16]: Check

File: Tailspin.SpaceGame.Web/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -12,8 +12,8 @@
 </style>
 <section class="intro">
     <div class="container">
-        <img class="title" src="/images/space-game-title.svg" alt="Space Game Adventure">
```
Comment (vaishnav-canarys) [inline Tailspin.SpaceGame.Web/Views/Home/Index.cshtml:15]: check

**Review by sriplayground — APPROVED** [review]

File: Tailspin.SpaceGame.Web/Views/Home/Index.cshtml (hunk 1 above)
Comment (sriplayground) [reply Tailspin.SpaceGame.Web/Views/Home/Index.cshtml:16]: The changes looks good!

Comment (vaishnav-canarys) [reply Tailspin.SpaceGame.Web/Views/Home/Index.cshtml:16]: changes ok

**Pull request merged**